### ISSUE TITLE
Total build time for matrix builds is sum of job times

### ISSUE THREAD

**Issue by bmaher** [travis-ci/travis-ci#8731]
The total build time (shown in the sidebar) for a build configured to have parallel matrix jobs, currently shows the time as a sum of the time of all jobs.

E.g. I have two jobs running that take 5min each, but run in parallel, the total build time will be shown as 10min.

I would expect the total build time to show the time of the longest job.

**Comment by CliffordSimak** [comment]
You can write `travis_wait 15 your_command`

More about that https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

**Comment by bmaher** [comment]
Hi Alexander,

Thanks for the tip, however unfortunately it’s not related to the issue.
The issue is regarding the displayed time in the Travis UI.
The time displayed is a sum of the time of all jobs, rather than the time of the longest job.
This means that even if two parallel jobs completed in a max of 5min, the UI will say that the build took 10min (sum of both jobs).

Attached are a couple of screenshots showing the problem.
In ‘correct_times.png’ you can see that the build time is shown correctly (the longest job time).
In ‘incorrect_times.png’ you can see that on the overview, the time is actually a sum of the time for both sub-jobs.
It looks like it’s displaying the ‘Total time’ value which is misleading for parallel jobs.

Thanks,

Brian

**Comment by BanzaiMan** [comment]
Duplicate of #1486.
